src/app_service/service/community_tokens/dto/community_token.nim

```diff
@@ -10,6 +10,13 @@ type
     InProgress,
     Deployed
 
+# determines what is the type of the token: owner, master or normal community contract
+type
+  PrivilegesLevel* {.pure.} = enum
+    Owner,
+    Master,
```

Comment:
Again, please name this `TokenMaster` instead

Reply:
PrivilegesLevel is called in status-go OwnerLevel, MasterLevel, CommunityLevel.
Why not TokenOwner, but TokenMaster ?

Reply:
> PrivilegesLevel is called in status-go OwnerLevel, MasterLevel, CommunityLevel.

I think it should be called `TokenMasterLevel` then, for the same reason above.

> Why not TokenOwner, but TokenMaster ?

Owner is a role, but Master isn't, it's TokenMaster.